zombies.js

```diff
@@ -79,6 +89,26 @@
  * @property {method} getMaxHealth         Returns private variable `maxHealth`.
  */
 
+function Player(name, health, strength, speed) {
+  this._pack = [];
```

Comment:
putting `_pack` and `_maxHealth` on this makes it public. Just do:
`const _pack = [];` to make it private. Same for max health. You will need to define both the `this.getPack()` and `this.getMaxHealth()` within the constructor to create function that allow you access to those variables.